testing/upload_validation.ts

```diff
@@ -91,6 +91,12 @@ import * as z from 'zod';
  * a real RegExp object.
  */
 export const PACKS_VALID_COLUMN_FORMAT_MATCHER_REGEX = /^\/(.*)\/([a-z]+)?$/;
+export const BUILDING_BLOCK_COUNT_LIMIT = 500;
+export const BUILDING_BLOCK_NAME_CHARACTER_LIMIT = 50;
+export const BUILDING_BLOCK_DESCRIPTION_CHARACTER_LIMIT = 500;
+export const COLUMN_MATCHERS_CHARACTER_LIMIT = 100;
+export const COLUMN_MATCHERS_COUNT_LIMIT = 10;
+export const NETWORK_DOMAIN_CHARACTER_LIMIT = 100;
 
```

Comment:
I came up with random values for these. Open to discussing and changing them

Reply:
Apparently domains are officially limited to 253 characters, maybe we could just go with that?